src/main/java/org/gridsuite/merge/orchestrator/server/repositories/IgmRepository.java

```diff
@@ -19,25 +20,18 @@
  * @author Franck Lecuyer <franck.lecuyer at rte-france.com>
  */
 @Repository
-public interface IgmRepository extends CassandraRepository<IgmEntity, IgmEntityKey> {
+public interface IgmRepository extends JpaRepository<IgmEntity, IgmEntityKey> {
 
-    @Query("SELECT * FROM merge_igm WHERE processUuid = :processUuid")
+    @Query(value = "SELECT igm from #{#entityName} as igm where igm.key.processUuid = :processUuid")
```

Comment:
Really  necessary ?

Reply:
primary key is composite key

Reply:
Yes and Spring Data JPA use also the query derivd by name fot the composite keys :
`@Repository
public interface IgmRepository extends JpaRepository<IgmEntity, IgmEntityKey> {
    
    List<IgmEntity> findByKeyProcessUuid(UUID processUuid);
    
    List<IgmEntity> findByKeyProcessUuidAndKeyDate(UUID processUuid, LocalDateTime date);

    @Modifying
    void deleteByKeyProcessUuid(UUID processUuid);
    
    Optional<IgmEntity> findByKeyProcessUuidAndKeyDateAndKeyTso(UUID processUuid, LocalDateTime date, String tso);
}`

Reply:
Done in PR for migration to PostGres : https://github.com/gridsuite/merge-orchestrator-server/pull/50